pkg/download/bucket/testdata/buckets.json

```diff
@@ -6,7 +6,7 @@
             "displayName": "Default metrics (15 months)",
             "status": "active",
             "retentionDays": 462,
-            "metricInterval": "PT1M",
+            "metricInterval": "{{.PT1M}}",
```

Comment:
Is this example purely fictitious? To show that something has `{{` ... `}}` will be escaped? Or is this something that actually happens?

Reply:
It's fictional, to test if the escaping works.
At the moment, there isn't a way to set something like `{.something}` except fot the displayName. The other (optional) properties don't allow that